br/pkg/task/stream.go

```diff
@@ -1321,6 +1321,9 @@ func restoreStream(
 	ctx, cancelFn := context.WithCancel(c)
 	defer cancelFn()
 
+	restoreCfg := enableTiDBConfig()
```

Comment:
maybe let's put a comment here for what it's used for?

Reply:
I renamed the function. Perhaps its name now presents its intention properly?